lib/kitchen/provisioner/chef/bourne_shell.rb

```diff
@@ -0,0 +1,107 @@
+# -*- encoding: utf-8 -*-
+#
+# Author:: Matt Wrock (<matt@mattwrock.com>)
+#
+# Copyright (C) 2014, Matt Wrock
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Kitchen
+
+  module Provisioner
+
+    module Chef
+
+      # Provides commands for interacting with chef on a test instance
+      #
+      # @author Matt Wrock <matt@mattwrock.com>
+      module BourneShell
+
+        # The path to the root of a chef install
+        #
+        # @return [String] absolute path to chef root
+        def chef_omnibus_root
+          "/opt/chef"
+        end
+
+        # path to chef-solo relative to the chef_omnibus_root
+        #
+        # @return [String] path to chef solo
+        def chef_solo_file
+          "bin/chef-solo"
+        end
+
+        # path to chef-client relative to the chef_omnibus_root
+        #
+        # @return [String] path to chef client
+        def chef_client_file
+          "bin/chef-client"
+        end
+
+        # A command for initializing an instance for working with chef
+        # the calling provisioner may have a omnibus root configured by
+        # the user that is different from this module's chef_omnibus_root.
+        #
+        # @param root_path [String] root path to chef omnibus
+        # @return [String] command to run on instance initializing
+        # chef environment
+        def init_command(root_path)
+          dirs = %w[cookbooks data data_bags environments roles clients].
+            map { |dir| File.join(root_path, dir) }.join(" ")
+          lines = ["#{sudo("rm")} -rf #{dirs}", "mkdir -p #{root_path}"]
```

Comment:
I think it is in this case. This is pretty much a copy from https://github.com/test-kitchen/test-kitchen/blob/master/lib/kitchen/provisioner/chef_base.rb#L96